modules/aggregation-layers/src/utils/gpu-grid-aggregation/gpu-grid-aggregator.js

```diff
@@ -968,6 +972,16 @@ export default class GPUGridAggregator {
   }
   /* eslint-enable complexity, max-depth */
 
+  _getMinMaxTexture(name, weights, id) {
+    if (weights[name]) {
+      return weights[name];
+    }
+    const resourceName = `${id}-name`;
+    const {resources} = this.state;
+    resources[resourceName] = getFloatTexture(this.gl, {id: `resourceName`});
```

Comment:
Is this code path being deprecated in favor of externally created textures? Generally speaking, I'd prefer a single owner of the resource instances.

At least document what the `weight` object expects?

Reply:
Original intention was not do `auto resource creation`, hence texture is updated only when provided by the user. But looking more into this, `Textures` are always need to GPU Aggregation path (for setting up Framebuffers), so it is better aggregator own these resources. Update the code and docs accordingly. We can do a similar pass for `Buffer` objects, but in a different PR.